build.sbt

```diff
@@ -259,6 +260,22 @@ lazy val lmIvy = (project in file("ivy"))
     ),
   )
 
+lazy val lmCoursier = (project in file("coursier"))
+  .enablePlugins(ContrabandPlugin, JsonCodecPlugin)
```

Comment:
Are these plugins necessary?
```suggestion
```

Reply:
kept Contraband that is now used and removed the unused JsonCodec